.github/workflows/self-approval.yml

```diff
@@ -0,0 +1,43 @@
+# Flow for self-approving the PRs, used by maintainers for cases aligned between them.
+# To trigger it, maintainer need to commit and push:
+# $ git commit -m "self-approval" --allow-empty
+
+name: self-approval
+
+on:
+  - pull_request_target
+
+permissions:
+  pull-requests: write
+  contents: write
+
+jobs:
+  job:
+    runs-on: ubuntu-latest
+    if: ${{ contains(fromJson('["julienfalque", "keradus", "keradus-ci", "kubawerlos", "localheinz", "Wirone"]'), github.actor) }}
+    env:
+      PR_URL: ${{github.event.pull_request.html_url}}
+      GITHUB_TOKEN: ${{secrets.GITHUB_TOKEN}}
+      MARKER: "self-approval"
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+        with:
+          ref: ${{ github.event.after }}
+      - name: get marker
+        id: marker
+        run: |
+          msg=$(git show -s --format=%s ${{ github.event.after }})
+          echo $msg
+          indicator=$([[ $msg == $MARKER ]] && echo yes || echo no)
+          echo $indicator
+          echo "indicator_value=${indicator}" >> "$GITHUB_OUTPUT"
+      - name: skip
+        if: steps.marker.outputs.indicator_value != 'yes'
+        run: echo no marker detected, skipping
```

Comment:
I am not sure we need this. Personally I would keep only triggered path, if `indicator_value` is not `yes`, then the next step will be skipped and it is enough to see that conditions were not met.

```suggestion
```

Reply:
yeah, i used it for debugging mostly :D